packer/vagrant.json

```diff
@@ -0,0 +1,142 @@
+{
```

Comment:
As discussed, since we can't have comments in this Json file, shall we put a brief outline of what each bit of this file does in the readme

Reply:
added to packer/README.md

Reply:
Looks good.